test/Microsoft.Framework.Runtime.FunctionalTests/ServiceBreadcrumbsFacts.cs

```diff
@@ -34,7 +35,8 @@ public void BreadcrumbsAreCreatedSuccessfully()
             Assert.True(File.Exists(Path.Combine(TempFolderPath, "Test.1.0.0")));
         }
 
-        [Fact]
+        [ConditionalFact]
+        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
```

Comment:
I think we still need the coverage on Mono. Can we fix it by using `System.Environment.UserName` or something similar?